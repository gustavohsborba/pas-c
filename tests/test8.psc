var
    n, c, r is int;

begin
    out({Digite um numero: });
    in(n);
    b = 10;

    c:=1;
    do
        out(n);
        out({ x });
        out(c);
        out({ = });
        out(n*c);
        out({\n});
    while c !=10;

end.