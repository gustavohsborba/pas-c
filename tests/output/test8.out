Opening file tests/test8.psc...
Token found: (TOK_VAR)
Token found: (TOK_ID, n)
Token found: (TOK_COMMA)
Token found: (TOK_ID, c)
Token found: (TOK_COMMA)
Token found: (TOK_ID, r)
Token found: (TOK_IS)
Token found: (TOK_INT)
Token found: (TOK_SEMICOLON)
Token found: (TOK_BEGIN)
Token found: (TOK_OUT)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_CONST_STR, Digite um numero: )
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_IN)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_ID, n)
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_ID, c)
Token found: (TOK_ASSIGN)
Token found: (TOK_CONST_INT, 1)
Token found: (TOK_SEMICOLON)
Token found: (TOK_DO)
Token found: (TOK_OUT)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_ID, n)
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_OUT)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_CONST_STR,  x )
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_OUT)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_ID, c)
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_OUT)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_CONST_STR,  = )
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_OUT)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_ID, n)
Token found: (TOK_MULT)
Token found: (TOK_ID, c)
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_OUT)
Token found: (TOK_PAR_OPEN)
Token found: (TOK_CONST_STR, \n)
Token found: (TOK_PAR_CLOSE)
Token found: (TOK_SEMICOLON)
Token found: (TOK_WHILE)
Token found: (TOK_ID, c)
ERROR in line 16, column 13: Token ! is not an operator
Token found: (TOK_EQUALS)
Token found: (TOK_CONST_INT, 10)
Token found: (TOK_SEMICOLON)
Token found: (TOK_END)
ERROR in line 18, column 5: Token . is not an operator
Token found: (TOK_EOF)
